docs/book/installation/installation.rst

```diff
@@ -64,7 +64,11 @@ After everything is in place, run the following command to install Sylius:
 Configuring Mailer
 ------------------
 
-In order to send emails you need to configure Mailer Service. Basically you need to:
+In order to send emails you need to configure Mailer Service. Basically there are multiple ways to do it:
+
+* We are recommending to use `Symfony Mailer <https://symfony.com/doc/current/mailer.html>`_
+* In Symfony Mailer use the `3rd Party Transports <https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport>`_
+* Use your own mailer service with this short configuration:
 
 1. **Create an account on a mailing service.**
 2. **In your** ``.env`` **file modify the** ``MAILER_URL`` **variable.**
```

Comment:
Can we duplicate this fragment below `* We are recommending to use Symfony Mailer` with the change `MAILER_URL` to `MAILER_DSN`